grails-app/views/public/show.gsp

```diff
@@ -0,0 +1,665 @@
+collectionsUrl: "${grailsApplication.config.grails.serverURL}",
+                /* base url of the biocache ws*/
+                biocacheServicesUrl: biocacheServicesUrl,
+                /* base url of the biocache webapp*/
+                biocacheWebappUrl: biocacheWebappUrl,
+                /* a uid or list of uids to chart - either this or query must be present
+                   (unless the facet data is passed in directly AND clickThru is set to false) */
+                instanceUid: "${instance.uid}",
+                /* the list of charts to be drawn (these are specified in the one call because a single request can get the data for all of them) */
+                charts: ['country','state','species_group','assertions','type_status',
+                'biogeographic_region','data_resource_uid','state_conservation','occurrence_year']
+                    /* override default options for individual charts */
+        }
+var taxonomyChartOptions = {
+    /* base url of the collectory */
+collectionsUrl: "${grailsApplication.config.grails.serverURL}",
+                /* base url of the biocache */
+                biocacheServicesUrl: biocacheServicesUrl,
+                /* base url of the biocache webapp*/
+                biocacheWebappUrl: biocacheWebappUrl,
+                /* support drill down into chart - default is true */
+                drillDown: true,
+                /* a uid or list of uids to chart - either this or query must be present */
+                instanceUid: "${instance.uid}",
+                /* threshold value to use for automagic rank selection - defaults to 55 */
+                threshold: 55,
+                rank: "${instance.startingRankHint()}"
+}
+/************************************************************\
+ *
+ \************************************************************/
+var queryString = '';
+var decadeUrl = '';
+var initial = -120;
+var imageWidth = 240;
+var eachPercent = (imageWidth/2)/100;
+
+$('img#mapLegend').each(function(i, n) {
+         // if legend doesn't load, then it must be a point map
+          $(this).error(function() {
+                    $(this).attr('src',"${resource(dir:'images/map',file:'single-occurrences.png')}");
+                  });
+          // IE hack as IE doesn't trigger the error handler
+          /*if ($.browser.msie && !n.complete) {
+                $(this).attr('src',"${resource(dir:'images/map',file:'single-occurrences.png')}");
+              }*/
+        });
+/************************************************************\
+ * initiate ajax calls
+ \************************************************************/
+function onLoadCallback() {
+
+    if(loadLoggerStats){
+        loadDownloadStats("${grailsApplication.config.loggerURL}", "${instance.uid}","${instance.name}", "1002");
+    }
+
+    // records
+    $.ajax({
+url: urlConcat(biocacheServicesUrl, "/occurrences/search.json?pageSize=0&q=") + buildQueryString("${instance.uid}"),
```

Comment:
It's all over the place. Should re-indent. I didn't touch the parts I didn't absolutely had to.